server/authn/jwt.go

```diff
@@ -106,9 +106,20 @@ func SpiffeIDFromContext(ctx context.Context) (spiffeid.ID, bool) {
 	return id, ok
 }
 
+// Health check endpoints that should bypass authentication.
+var healthCheckEndpoints = map[string]bool{
+	"/grpc.health.v1.Health/Check": true,
+	"/grpc.health.v1.Health/Watch": true,
+}
+
```

Comment:
Consider add these to the `healthcheck` package as constants with a utility function to check path and import it 
`import "github.com/agntcy/dir/server/healthcheck"`